docs/docs-content/clusters/cluster-management/cluster-updates.md

```diff
@@ -140,3 +142,123 @@ You can follow these steps to validate all cluster update approaches.
 3. Select the cluster you updated, and navigate to the **Profile** tab of the cluster.
 
 4. Confirm that the cluster is now using an updated profile.
+
+## Palette Reconciliation Behavior
+
+When cluster profile updates are applied, Palette will apply the changes to the cluster but will not delete any
+resources that are no longer in use. Depending on the situation, you can choose to keep or remove these resources. To
```

Comment:
```suggestion
resources that are no longer in use. You can choose to keep or remove these resources. To
```
"Depending on the situation" is kind of a filler phrase that you can do without here.

Reply:
That was part of my suggested change in all honesty ✋ .